src/app/public/modules/radio/radio.component.ts

```diff
@@ -91,15 +96,39 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     return this._disabled;
   }
 
+/**
+ * Specifies an ID for the radio button. If you do not specify an ID, an auto-incrementing integer generates unique IDs.
+ */
```

Comment:
Maybe a combo of the two so the default value is explained and less confusing?

An auto-incrementing unique integer. For example, `sky-radio-${++nextUniqueID}`

Reply:
Deleted my original comment because I read your suggestion too quickly. 

Would just having the description confuse users when they did see the integer? Basically, should we have both, description and example? I'm not sure which option would be more helpful for engineers and will lean on you for that one.

Reply:
Yeah, I like that. How about something like...
```
/**
 * Specifies an ID for the radio button.
@default is an auto-incrementing unique integer. For example: `sky-radio-1`.
 */
```
Not sure how that will render, but worth a shot?

Reply:
I mean, it's different but not the worst.